src/main/scala/com/databricks/labs/overwatch/pipeline/Gold.scala

```diff
@@ -310,7 +310,7 @@ class Gold(_workspace: Workspace, _database: Database, _config: Config)
   lazy private val appendNotebookCommandsFactProcess: () => ETLDefinition = {
     () =>
       ETLDefinition(
-        BronzeTargets.auditLogsTarget.asIncrementalDF(notebookCommandsFactModule, BronzeTargets.auditLogsTarget.incrementalColumns,1),
+        BronzeTargets.auditLogsTarget.asIncrementalDF(notebookCommandsFactModule, BronzeTargets.auditLogsTarget.incrementalColumns,30),
```

Comment:
How does this affect the performance of this table? There are potentially many rows in Audit logs that would be brought in when using a 30 day lag